### DISCUSSION TITLE
На Mintmanga наехал РКН

### DISCUSSION THREAD

**Discussion by mark3701** [lirix360/ReadmangaGrabber#98] [Q&A]
Здравствуйте,
На Mintmanga наехал РКН и у них там переезд на новый сайт https://seimanga.me/, поэтому Manga Grabber не хочет качать с нового сайта. Буду очень признателен за расширение вашей замечательной утилиты!!! Жду с нетерпением!

**Comment by lirix360** [comment]
> Здравствуйте, На Mintmanga наехал РКН и у них там переезд на новый сайт https://seimanga.me/, поэтому Manga Grabber не хочет качать с нового сайта. Буду очень признателен за расширение вашей замечательной утилиты!!! Жду с нетерпением!

Seimanga работает.

**Reply by mark3701** [reply]
Спасибо!
